.eslintignore

```diff
@@ -1,5 +1,9 @@
-**/node_modules/*
-**/dist/*
-android/**/build/*
-.github/actions/**/index.js"
+.*
```

Comment:
*question:* does this line ignore anything that starts with `.`? If yes, great change! I was going to ask that we ignore `.bundle` and `.expo` too but it seems like that's covered.

Reply:
Previously we had `.eslintignore` file and `ignorePatterns` in eslint config:
```ts
ignorePatterns: ['!.*', 'src/vendor', '.github/actions/**/index.js', 'desktop/dist/*.js', 'dist/*.js', 'node_modules/.bin/**', 'node_modules/.cache/**', '.git/**'],
```

I removed ignorePatterns from the config and unified all patterns. So nothing should really change 👍

Reply:
Yes! I thought it was clever but turns out here are two caveats:
- With this approach `.github` and `.storybook` weren't linted (it's more complicated because we want to lint `.github` but not `.github/actions/**/index.js`)
- Turns out dot files/directories are automatically omitted so no need for `.*`, instead I did this:
```
!.storybook
!.github
```

Reply:
@parasharrajat I checked it myself but could you double check that correct directories are linted (.github, .storybook, src, web, desktop etc) and correct are omitted (.github/actions/**/index.js, configs, dist, .expo, .git)? Thank you!